### PR TITLE
Add body force back in

### LINKED ISSUE #9528: Add Navier-Stokes body force term back in
### Description of the enhancement or error report
The gravity body force term in `INSMomentumBase` is commented out. The `gravity` parameter does _nothing_.

### Rationale for the enhancement or information for reproducing the error
Add useful term back in and give the unused parameter a new purpose in life.

### Identified impact
Added capability

### PR DESCRIPTION
**Pull request by dschwen**
Closes #9528

### CODE REVIEW

**Comment by jwpeterson** [comment]
Looks like you are in conflict (lowers shades) with yourself.

**Comment by jwpeterson** [comment]
Wait, why all the `const` additions? Seems extraneous to this PR, should at the very least be in a separate commit.

**Comment by dschwen** [comment]
So that first clang-format failure was the result of clang-format needing two iterations to reach a stable configuration.

File: modules/navier_stokes/src/kernels/INSMomentumBase.C [hunk 1]
```diff
@@ -97,12 +97,10 @@ INSMomentumBase::computeQpResidual()
   // Call derived class to compute the viscous contribution.
   Real viscous_part = computeQpResidualViscousPart();
 
-  // Body force term.  For truly incompressible flow, this term is constant, and
-  // since it is proportional to g, can be written as the gradient of some scalar
-  // and absorbed into the pressure definition.
-  // Real body_force_part = - _rho[_qp] * _gravity(_component);
+  // Body force term
+  const Real body_force_part = -_rho[_qp] * _gravity(_component);
```
Comment (jwpeterson) [inline modules/navier_stokes/src/kernels/INSMomentumBase.C]: I'd do a fixup commit on this one multiplying by the test function...

**Review by jwpeterson — CHANGES_REQUESTED** [review]
Need to fix bug in body force residual contribution.

**Comment by jwpeterson** [comment]
Oh, and we need to add the hydrostatic test case that exercises the term you uncommented...

**Review by jwpeterson — APPROVED** [review]

**Pull request merged**